src/services/blocks/BlocksService.spec.ts

```diff
@@ -309,23 +307,43 @@ describe('BlocksService', () => {
 			// Reset LRU cache
 			cache.reset();
 
-			const historicApi = createApiWithAugmentations(
-				polkadotMetadataV29.toHex()
-			) as ApiDecoration<'promise'>;
+			/**
+			 * This is the mockApi adjusted to mock a runtime that uses
+			 * consts.system.blockWeights for its weight nomination.
+			 */
+			const mockHistoricApiAdjusted = {
```

Comment:
I not sure what are you trying to do here. is this a unit test or an integration test? I am confused by the casting concatenation. Why not create a mockClass of `ApiDecoration` with the data that you need? We have `jest` in place we can take advantage of it

Reply:
Via our conversation going to leave as is. But this is a good point to think about potentially switching over all the mock apis to a jest mockApi.